src/archivematicaCommon/lib/storageService.py

```diff
@@ -393,58 +354,60 @@ def request_reingest(package_uuid, reingest_type, processing_config):
-        LOGGER.exception("Unable to reingest {}".format(package_uuid))
-        try:
-            return e.response.json()
-        except Exception:
-            return {'error': True}
+        response = _storage_api_session().post(url, json=api_request)
     except requests.ConnectionError as e:
         LOGGER.exception("Could not connect to storage service")
         return {'error': True, 'message': 'Could not connect to storage service'}
-    return response
+    except requests.exceptions.RequestException as e:
+        LOGGER.exception("Unable to reingest %s", package_uuid)
+        try:
+            return response.json()
+        except Exception:
+            return {'error': True}
+    return response.json()
 
 
 def request_file_deletion(uuid, user_id, user_email, reason_for_deletion):
     """ Returns the server response. """
 
-    api = _storage_api()
     api_request = {
         'event_reason': reason_for_deletion,
-        'pipeline':     get_setting('dashboard_uuid'),
-        'user_email':   user_email,
-        'user_id':      user_id
+        'pipeline': get_setting('dashboard_uuid'),
+        'user_email': user_email,
+        'user_id': user_id,
     }
-
-    return api.file(uuid).delete_aip.post(api_request)
-
+    url = _storage_service_url() + 'file/' + uuid + '/delete_aip/'
+    response = _storage_api_session().post(url, json=api_request)
+    return response.json()
 
 def post_store_aip_callback(uuid):
-    api = _storage_api()
-    return api.file(uuid).send_callback.post_store.get()
+    url = _storage_service_url() + 'file/' + uuid + '/send_callback/post_store/'
+    response = _storage_api_session().get(url)
+    try:
+        return response.json()
+    except Exception:
+        return response.text
 
 def get_file_metadata(**kwargs):
-    api = _storage_api()
-    try:
-        return api.file.metadata.get(**kwargs)
-    except slumber.exceptions.HttpClientError:
+    url = _storage_service_url() + 'file/metadata/'
+    response = _storage_api_session().get(url, params=kwargs)
+    if 400 <= response.status_code < 500:
         raise ResourceNotFound("No file found for arguments: {}".format(kwargs))
+    return response.json()
 
 def remove_files_from_transfer(transfer_uuid):
-    api = _storage_api()
-    api.file(transfer_uuid).contents.delete()
+    url = _storage_service_url() + 'file/' + transfer_uuid + '/contents/'
+    _storage_api_session().delete(url)
 
 def index_backlogged_transfer_contents(transfer_uuid, file_set):
-    api = _storage_api()
-    try:
-        api.file(transfer_uuid).contents.put(file_set)
-    except slumber.exceptions.HttpClientError as e:
-        raise BadRequest("Unable to add files to transfer: {}".format(e))
+    url = _storage_service_url() + 'file/' + transfer_uuid + '/contents/'
+    response = _storage_api_session().put(url, json=file_set)
+    if 400 <= response.status_code < 500:
```

Comment:
I was matching previous behaviour - `slumber.exceptions.HttpClientError` happens on 4xx responses but not 5xx responses.